src/components/ProgressBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+// Import react-circular-progressbar module and styles
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+const percentage = Math.floor(Math.random() * 20);
+
+const ProgressBar = () => (
+  <div style={{ width: 120, height: 120, marginRight: '20px' }}>
```

Comment:
- [ ] Great job on adding the progress bar :clap:. However, in React it is better practice if you want to use an inline style to use a variable. Using a variable to add inline style makes your React code cleaner and easy to maintain. So what I would suggest is to add your inline style like this :point_down: 
```suggestion
  const progressbarStyle = { width: 120, height: 120, marginRight: '20px' };
  <div style={progressbarStyle}>
```